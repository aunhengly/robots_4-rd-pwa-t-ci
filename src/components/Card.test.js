import { shallow, mount, render } from 'enzyme';
import renderer from 'react-test-renderer';

//mount is for full dom mounting
//render is render react component under
import React from 'react';
import Card from './Card';

it('expect it to render Card', () => {
   expect(shallow(<Card/>)).toMatchSnapshot();
})