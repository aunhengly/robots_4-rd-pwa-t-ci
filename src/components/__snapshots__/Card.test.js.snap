// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect it to render Card 1`] = `ShallowWrapper {}`;
